retworkx-core/src/centrality.rs

```diff
@@ -297,3 +294,71 @@ where
         sigma,
     }
 }
+
+/// Compute the closeness centrality of all nodes in a graph.
```

Comment:
Ideally we should also use here the extended docstring version you wrote in `__init__.py`.